### ISSUE TITLE
muffin-4.0.7 breaks some wine windows

### ISSUE THREAD

**Issue by artik** [linuxmint/muffin#501]
```
 * libmuffin0 4.0.7+tessa
 * Distribution - Mint 19.1
 * Nvidia GTX 1080 nvidia driver 430
 * 64 bit
 ```

**Issue**

Some Winehq (all versions) windows are not drawn properly anymore after an update from cinnamon-muffin 4.0.5 to 4.0.7. (screenshot attachment)

**Steps to reproduce**

Install Photoshop CS6, and display secondary toolbox windows

**Expected behaviour**

Proper display of the windows

**Other information**

![screenshot01 (1)](https://user-images.githubusercontent.com/1015856/57973148-8a18f680-79a4-11e9-8bfc-b8f57f2310d5.png)

**Comment by clefebvre** [comment]
Is there a free app we can test this with?

**Comment by clefebvre** [comment]
4.0.6 would be more or less the same as 4.0.5, it doesn't affect how things render, so this is pretty much a 4.0.7 regression.

**Comment by artik** [comment]
Yes, I checked the commit and was pretty sure the 4.0.7 is affected by the regression. The installer is free for 30 trial days.

I'm going to look for a free app that could also be affected, with no garantee of a result

**Comment by leigh123linux** [comment]
Are you sure it isn't a regression in nvidia driver 430?

**Comment by artik** [comment]
I tried on:

Wine 4.0, 4.8
Wine-staging 4.1, 4.7
Wine-devel latest version.

I also tried with:

Nvidia proprietary drivers 390, 410, 430
Nouveau open source drivers 1:1.0.15

To target the guilty, I reinstalled a fresh copy of Mint 19.1, and spent all the afternoon to update the packages one by one till I found it was muffin 4.0.7.

**Comment by jaszhix** [comment]
I can't reproduce this in Photoshop CC on master. There is one PR open for the maintenance branch that might fix this (#483) on 4.0.

![photoshop_cc](https://user-images.githubusercontent.com/6859057/58448163-97349480-80cc-11e9-8436-2c8c9783d6ec.png)

```
$ wine --version
wine-4.0.1
```

**Comment by jaszhix** [comment]
Update - I tested Photoshop CC on 4.0.7-maintenance, and not observing any difference in window rendering. Reverted back to 4.0.6. No difference, but the toolbar flickers black randomly, and the input is misaligned with the window.
